recipes/libbigwig/all/conanfile.py

```diff
@@ -26,32 +28,26 @@ class LibBigWigConan(ConanFile):
     default_options = {
         "shared": False,
         "fPIC": True,
-        "with_curl": False,
+        "with_curl": True,
         "with_zlibng": False
     }
 
     def configure(self):
         if self.options.shared:
-            del self.options.fPIC
-        try:
-            del self.settings.compiler.libcxx
-        except Exception:
-            pass
-        try:
-            del self.settings.compiler.cppstd
-        except Exception:
-            pass
+            self.options.rm_safe("fPIC")
+        self.settings.rm_safe("compiler.libcxx")
+        self.settings.rm_safe("compiler.cppstd")
 
     def layout(self):
         cmake_layout(self, src_folder="src")
 
     def requirements(self):
         if self.options.with_curl:
-            self.requires("libcurl/7.85.0")
+            self.requires("libcurl/8.0.1", transitive_headers=True)
```

Comment:
Sometimes it's hard to see where the need for this trait arises, so for posterity: Needed because of https://github.com/dpryan79/libBigWig/blob/master/bigWigIO.h#L5

Reply:
You're right, should've documented this with a comment in the recipe.

Thanks!

Reply:
That's a great way to do it yes :)

Reply:
Done!